src/org/traccar/Context.java

```diff
@@ -137,10 +137,13 @@ public static EventForwarder getEventForwarder() {
     public static void init(String[] arguments) throws Exception {
 
         config = new Config();
-        if (arguments.length > 0) {
-            config.load(arguments[0]);
+        if (arguments.length <= 0) {
+            System.out.println("No configuration file given.");
```

Comment:
Changes to this have nothing to do with registering new devices. I think you should exclude it.